.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Could you explain this file? this might be a mistake

Reply:
Yep, I will remove it. :)